### PR TITLE
App strings reworked 10

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: src/resources/i18n/translations/en.json [hunk 1]
```diff
@@ -604,7 +604,7 @@
   "Send us an email": "Send us an e-mail",
   "Setup": "Setup",
   "Settings": "Settings",
-  "Setup Wallet": "Set up wallet",
+  "Setup Wallet": "Set up",
```
Comment (comradekingu) [inline src/resources/i18n/translations/en.json:607]: Not sure

File: src/resources/i18n/translations/en.json [hunk 2]
```diff
@@ -712,9 +712,9 @@
   "Use custom web domain": "Custom web domain",
   "Use External Storage For Downloads": "To external storage card",
   "Username": "Username",
-  "value_tag_preview_boost": "A boost sends a donation.",
-  "value_tag_preview_boostagram": "A boostagram also adds a public message.",
-  "value_tag_preview_streaming": "Stream sends a donation for every minuted listened.",
+  "value_tag_preview_boost": "A <i>boost</i> sends a donation.",
+  "value_tag_preview_boostagram": "A <i>boostagram</i> also adds a public message.",
+  "value_tag_preview_streaming": "<i>Stream</i> sends a donation for every minuted listened.",
```
Comment (mitchdowney) [inline src/resources/i18n/translations/en.json:717]: It's fine if you want to italicize these, but this won't render correctly in the React Native Text component. Making this change would need it's own Github issue. I'll remove these after merging.

File: src/resources/i18n/translations/en.json [hunk 3]
```diff
@@ -742,26 +742,26 @@
   "Yes enable tracking": "Yes, track me",
   "Yes save them": "Yes, save them",
   "Your premium membership is still valid": "Your premium membership is still valid. If you would ",
-  "You are about to be navigated to a website outside the app Are you sure you want to leave brandName": "Leave Podverse for a website outside the app?",
+  "You are about to be navigated to a website outside the app Are you sure you want to leave brandName": "Open website in web browser?",
```
Comment (mitchdowney) [inline src/resources/i18n/translations/en.json:745]: I was of the impression that we have to be explicit about when a user is "Leaving Podverse" for app store compliance. I'll merge this though and we'll see if they flag us.

**Review by mitchdowney — APPROVED** [review]

File: src/resources/i18n/translations/en.json (hunk 3 above)
Comment (comradekingu) [reply src/resources/i18n/translations/en.json:745]: When I click the GitHub link in "About" I am not actually leaving Podverse.
It just opens a web browser.

**Pull request merged**